tests/compare_scan_results.py

```diff
@@ -8,7 +8,7 @@
 from skyreader import SkyScanResult
 from wipac_dev_tools import logging_tools
 
-RTOL_PER_FIELD = {"llh": 0.18, "E_in": 0.7, "E_tot": 0.7}
+RTOL_PER_FIELD = {"llh": 0.22, "E_in": 0.7, "E_tot": 0.7}
```

Comment:
what are your thoughts on using different rtols for each reco-algo?

Reply:
It could be useful, although I am not sure this is really what we need.

Ideally we should be able to produce more "accurate" test data and have reasonably low tolerances, because with ~20% (~70% for energy reco) it's very easy to miss small changes.

I am afraid coarse scans are going to have bad/unstable likelihood values nevertheless, maybe we should "fabricate" test data according to the CI output instead of relying on Condor runs.

Reply:
By the way, we could get the test data out of CI "manually" by writing its JSON representation to stdout as part of the compare stage and using JSON for the reference files.

Reply:
Would it be possible to create a synthetic event that is fairly stable for coarse scans?  (or find one in real data, but that's by chance)  Then you could tighten down the tolerances, as it should always produce basically the same result.

Reply:
I have added a `cat` to print out the output JSON in CI for "realistic" scans. This can be used to extract the outputs from CI and replace the test data + tighten the tolerances.

Since now the compare stage is hardcoded to use `npz` I suggest we proceed with this change for all three recos after this PR is merged.

Reply:
Discuss this in #208.

Reply:
This is now fixed with updated test data.